torch/testing/_internal/common_quantization.py

```diff
@@ -631,6 +631,10 @@ def __init__(self, qconfig):
         self.bn2 = nn.BatchNorm3d(2).to(dtype=torch.float)
         self.relu3 = nn.ReLU(inplace=True).to(dtype=torch.float)
         self.conv3 = nn.Conv1d(3, 3, 2).to(dtype=torch.float)
+        if train:
```

Comment:
Can you instead check that the code raises an exception for Conv1d + Bn1d and remove the train flag for the model?